### ISSUE TITLE
Content Validator check throws non-existent error

### ISSUE THREAD

**Issue by wsjk** [spectacles-ci/spectacles#804]
Our Looker architecture is designed to have multiple projects that each analytics team controls and their projects gets imported into a central core project for cross team collaboration.

In each project we have a GHA that runs Spectables' Content Validator check. We also have a GHA in the Core project. 

All team projects contains generic lkml files with explore definitions they own, but these explore files aren't always included in the model file in the team project (eg `team.model`). Instead they will include them in the Core project (eg `core.model`) since it's available via import and allows for cross team collaboration.

In the cases where an explore *is* included in `team.model` file, we are getting CV errors for a user defined dash. The error is that the dash has tiles that references an explore that does not exist. The thing is that the explore does exist and it's included in `team.model` but not `core.model`. When we run CV in the Looker IDE, we do not see the errors from the Spectables CV check. 

Is the Spectables CV check somehow different than running CV in Looker browser?

**Comment by joshtemple** [comment]
It's running the same check as the browser content validator, but the Spectacles Looker user may have different permissions than the user you're viewing the browser content validator with.

You should sudo as the Spectacles Looker user and check the content validator again. You'll then want to double-check access grants, model permissions, etc.

**Comment by wsjk** [comment]
Hmm it's definitely not a permissions issue and the CV successfully passes some other checks before it hits this specific error I'm mentioning.

**Comment by DylanBaker** [comment]
@wsjk Do you mind sharing the exact command you're running? Wondering if there's something amiss there. Thanks!

**Comment by wsjk** [comment]
Here's the composite GHA we created referencing Spectacles to run in all our looker project repos

```
name: 'Looker Run Content Validator'
description: 'Runs Content Validator for current project and dev branch'
inputs:
  python_version:
    description: |
      Python Version
      Default is '3.10'.
    default: '3.10'
  LOOKER_CLIENT_ID:
    description: |
       Looker API Client ID (Needs Dev License)
  LOOKER_CLIENT_SECRET:
    description: |
      Looker API Client Secret (Needs Dev License)
  LOOKER_PROJECT:
    description: |
      name of Looker Project
  LOOKER_BRANCH:
    description: |
      name of Looker branch to checkout for Content Validator
    default: 'main'
runs:
  using: "composite"
  steps:
    - id: setup-python
      name: setup python
      uses: actions/setup-python@v4
      with:
        python-version: ${{ inputs.python_version }}
    - id: install-spectacles
      name: PIP Install Spectacles
      shell: bash
      run: |
        pip install spectacles==2.4.7
    - id: run-content-validator-ibotta-core
      name: Run Content Validator for ibotta_core project
      shell: bash
      run: |
        export LOOKER_BASE_URL=https://ibotta.looker.com
        export LOOKER_CLIENT_ID=${{ inputs.LOOKER_CLIENT_ID }}
        export LOOKER_CLIENT_SECRET=${{ inputs.LOOKER_CLIENT_SECRET }}
        
        spectacles content \
          --project ${{ inputs.LOOKER_PROJECT }} \
          --branch ${{ inputs.LOOKER_BRANCH }}  \
          --exclude-personal \
          --verbose
```

**Comment by DylanBaker** [comment]
@wsjk Could you try this with the commit instead of the branch? I'd be curious to see what happens then. How is the branch currently being passed through? I'm less familiar with how composites work in GHA.

**Comment by wsjk** [comment]
ah ok yea this is maybe the problem.

The GHAs in project repos that used the above composite GHA didnt provide branch arg. 

Thanks!

**Comment by DylanBaker** [comment]
Are you running with the `--remote-reset` flag? https://docs.spectacles.dev/cli/reference/content-validator/#remote-reset

**Comment by wsjk** [comment]
Hmm but i'm not sure why this would help. The CV error is that it can't find an explore being referenced in a dashboard even though it clearly exists. I feel like the remote reset would be more likely to cause error in case we created an explore in Looker IDE but didn't push to remote, right?

**Comment by wsjk** [comment]
The credentials we used for our Spectacles GHA was for a service acct we created that had high level developer permissions. Even when sudoing as user in Looker IDE and running Content Validator in browser, we saw no errors. Adding the `remote-reset` flag also did not solve the issue. When we upgraded permissions of the user to Admin is when we solved issue. Not sure why this was necessary, but that seems to have solved the issue.
